web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx

```diff
@@ -80,8 +79,8 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
     },
     {
       title: "Created By",
-      dataIndex: "createdById",
-      key: "createdById",
+      dataIndex: "createdBy",
+      key: "createdBy",
```

Comment:
Sorry nothing about this point. But in the render for Action please fix the DownloadButton. Its doing the <DownloadButton></DownloadButton> thing

Reply:
Can you explain more please?

Reply:
Sorry not sure what happened to my comment... 
What I meant is in this component's render you are using the DownloadButton component, right? Its written like <DownloadButton></DownloadButton>, which we want to avoid

Reply:
Done!